src/App.tsx

```diff
@@ -145,29 +207,17 @@ export const App: React.FC = () => {
 
```

Comment:
You should hide the footer if there are no todos. And you should add a counter for items left.